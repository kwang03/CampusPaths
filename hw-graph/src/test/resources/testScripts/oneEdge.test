#Tests the behavior of a graph with just one edge between two nodes

#Creates graph
CreateGraph myGraph

#Adds nodes
AddNode myGraph n1
AddNode myGraph n2

#Adds edge
AddEdge myGraph n1 n2 e12

#Lists children
ListChildren myGraph n1
ListChildren myGraph n2

#Lists Nodes
ListNodes myGraph