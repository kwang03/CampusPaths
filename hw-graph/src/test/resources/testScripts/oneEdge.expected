#Tests the behavior of a graph with just one edge between two nodes

#Creates graph
created graph myGraph

#Adds nodes
added node n1 to myGraph
added node n2 to myGraph

#Adds edge
added edge e12 from n1 to n2 in myGraph

#Lists children
the children of n1 in myGraph are: n2(e12)
the children of n2 in myGraph are:

#Lists Nodes
myGraph contains: n1 n2